ChangeLog

```diff
@@ -1,3 +1,92 @@
+3.16.0:
+	- Added `cf-secret` binary for host-specific encryption (CFE-2613)
+	- 'cf-check diagnose --test-write' can now be used to test writing
+	  into LMDB files (ENT-4484)
+	- 'if' constraint now works in combination with class contexts
+	  (CFE-2615)
+	- Added $(sys.cf_version_release) variable (ENT-5348)
+	- Added new macros to parser: else, maximum_version, between_versions, 
+	  before_version, at_version and after_version (CFE-3198)
+	- Added cf-postgres requirement to cf-apache and cf-hub systemd units
+	  (ENT-5125)
+	- Added files promise content attribute (CFE-3276)
+	- Added string_trim() policy function (CFE-3074)
+	- Added warning if CSV parser parses nothing from non-empty file
+	  (CFE-3256)
+	- All changes made by 'files' promises are now reported (ENT-5291)
+	- Bootstrap to loopback interface is now allowed, with a warning
+	  (CFE-3304)
+	- Client initiated reporting was fixed on RHEL 8.1 (ENT-5415)
+	- Directory and file creations are now properly reported as 'info'
```

Comment:
Merge this one into the one above about all files promise changes being reported?